app/services/bulk_upload/lettings/year2023/row_parser.rb

```diff
@@ -383,6 +383,8 @@ class BulkUpload::Lettings::Year2023::RowParser
 
   validate :validate_declaration_acceptance, on: :after_log
 
+  validate :validate_all_charges_given, on: :after_log, if: proc { is_carehome.zero? }
```

Comment:
Wanna run this before validating nulls, cause validate_nulls would add an extra error otherwise